internetarchive/cli/ia_metadata.py

```diff
@@ -226,15 +232,16 @@ def main(argv, session):
             else:
                 responses.append(modify_metadata(item, metadata, args))
             if (i + 1) == len(args['<identifier>']):
-                if all(r.status_code == 200 for r in responses):
+                if all(cast(Response, r).status_code == 200 for r in responses):
```

Comment:
I'm not too fond of this cast, since it is really strange for the readability. But I guess `mypy` strictly demands it.

Reply:
You are right.  I removed it in favor of a `# type: ignore` which is easier on the reader.